apps/dashboard/app/models/script.rb

```diff
@@ -176,6 +192,18 @@ def most_recent_job_cluster
 
   private
 
+  def script_file
+    File.join(Script.scripts_dir(project_dir), "#{id}.yml")
+  end
+
+  def output_dir
```

Comment:
Let's save `output_dir` for later because I figure something like that'll be configurable at the project level? Or at least log location will be configurable at some level.

Reply:
OK, will save for later.

At the moment, I am not sure how I can delete the output files generated by the script as these are dependants on the script or the scheduler. Do you have any suggestions?

Reply:
> At the moment, I am not sure how I can delete the output files generated by the script as these are dependants on the script or the scheduler. Do you have any suggestions?

I suspect that's why we punted on deleting. If you think of a real project, there would be source code + input data + output data + lots more. If we don't wipe the whole directory, we should really just wipe what _we_ created.

Which is to say - leave the logs there for now.

Reply:
OK - will leave for now